src/services/cli-http-client.js

```diff
@@ -80,20 +89,20 @@ class CliRequestClient {
     try {
       const response = await this.http(options);
 
-      this.logger.debug('response.statusCode: ' + response.statusCode);
+      this.logger.debug('response.statusCode: ' + response.status);
       this.logger.debug('response.headers: ' + JSON.stringify(response.headers));
 
-      if (response.statusCode < 200 || response.statusCode >= 300) {
-        const parsed = JSON.parse(response.body);
-        throw new TwilioCliError(`Error code ${parsed.code} from Twilio: ${parsed.message}. See ${parsed.more_info} for more info.`, parsed.code);
-      }
-
       return response;
```

Comment:
Prefer to wrap this in a new object to avoid the downstream renames. E.g.,
```
return {
  body: response.data,
  statusCode: response.status,
  headers: response.headers
};
```